src/main/webapp/WEB-INF/jsp/scripts.jsp

```diff
@@ -18,10 +18,19 @@
     under the License.
 
 --%>
+
```

Comment:
Hello Benito,

I made that change because I thought tooltip() was getting reassigned back to jquery ui, as at the moment any tooltip() invocation seems to fail within Manchester University’s implementation of the newsreader portlet within uPortal 4.2, and I thought it was because the following was happening:

jqueryUI is assigned the tooltip() method
Then bootstrap tries to assign itself the tooltip() method – which causes an issue
Then this portlet, as it includes it’s own version of jquery ui tries to assign itself the tooltip() method

Hence tooltip() would not work for either bootstrap, or jqueryUI on hover, which is why I thought the following lines of code were required after jquery UI was included again.

$.widget.bridge('uibutton', $.ui.button);





$.widget.bridge('uitooltip', $.ui.uitooltip);


I can not see your pull request, so I can not comment about the changes you have made, however I imagine you removed any css targeted at ul.newstories/ul.feed and instead targeted .feed/.news-stories.

Kindly,
Michael.


From: Benito Gonzalez [mailto:notifications@github.com]
Sent: 29 March 2016 06:34
To: Jasig/NewsReaderPortlet
Cc: Michael Vose
Subject: Re: [Jasig/NewsReaderPortlet] MMFS-644 changed handlebars code around. (#29)


In src/main/webapp/WEB-INF/jsp/scripts.jsp<https://github.com/Jasig/NewsReaderPortlet/pull/29#discussion_r57673720>:

> @@ -18,10 +18,19 @@

>      under the License.

>

>  --%>

> +

I found that the console errors were caused by resizing the window. The resize() function was using a call that is not valid for this plugin.

Were there other reasons for this change? I do not see any errors when I removed the erroneous line.

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/Jasig/NewsReaderPortlet/pull/29/files/31ddfadf6ab7fd2a951047d72616a362b7ea44f2#r57673720>